openreview/conference/helpers.py

```diff
@@ -607,15 +607,18 @@ def get_bid_stages(request_forum):
     else:
         bid_due_date = None
 
-    reviewer_bid_stage = openreview.stages.BidStage(request_forum.content['venue_id'] + '/Reviewers', start_date = bid_start_date, due_date = bid_due_date, request_count = int(request_forum.content.get('bid_count', 50)))
+    reviewers_name = venue.reviewers_name if venue else 'Reviewers'
+    reviewer_bid_stage = openreview.stages.BidStage(request_forum.content['venue_id'] + f'/{reviewers_name}', start_date = bid_start_date, due_date = bid_due_date, request_count = int(request_forum.content.get('bid_count', 50)))
```

Comment:
can we figure out the reviewers_name inside of the BidStage?

for example:

if committee_id == venue.reviewers_id:
       commmittee_name = reviewers_name

Reply:
Where would `committee_id` come from in this function?

Reply:
from here? https://github.com/openreview/openreview-py/blob/master/openreview/stages/venue_stages.py#L382

Reply:
ohhhh I see, you are building the BidStage parameters. 

In this case, I would do:

```
reviewer_bid_stage = openreview.stages.BidStage(reviewers_id, start_date = bid_start_date, due_date = bid_due_date, request_count = int(request_forum.content.get('bid_count', 50)))
```

and pass reviewers_id as a param instead of the venue object.

Reply:
same for area chairs, if area_chairs_id is None then you don't set the stage.

Reply:
To get the reviewers_id don't I still need the venue object? To access `venue.get_reviewers_id()`?

Reply:
yes but I was thinking that can pass the reviewers_id directly instead of passing the venue object because this call is being done from the API 1 conference object too.

```
get_bid_stages(request_forum, reviewers_id, area_chairs_id, senior_area_chairs_id):
```

Reply:
does this mean for API 1 support I should still use the request form?:
```
reviewer_bid_stage = openreview.stages.BidStage(
        reviewers_id if reviewers_id else request_forum.content['venue_id'] + '/Reviewers', 
        ...
```

Reply:
no, the API 1 can pass these parameters right? `reviewers_id, area_chairs_id, senior_area_chairs_id`?

Reply:
ah that makes sense now, I thought I shouldn't modify API 1 code. Fixed now.